pytket/tests/architecture_aware_synthesis_test.py

```diff
@@ -202,6 +202,15 @@ def test_noncontiguous_arc_phase_poly() -> None:
     assert c.n_gates_of_type(OpType.CX) == 0
 
 
+def test_compose_ppb() -> None:
+    circ = Circuit(5).CZ(0, 1).CZ(1, 2).CX(2, 3).CX(3, 4)
+    pass1 = ComposePhasePolyBoxes(min_size=2)
+    cu = CompilationUnit(circ)
+    assert pass1.apply(cu)
+    out_circ = cu.circuit
+    assert out_circ.depth() == 6
+
```

Comment:
I notice that the phase poly box at the end of `out_circ` spans all 5 qubits, but when decomposed it only has CXs on qubits 2, 3 and 4. Is this expected? Having the box span qubits that it does not affect could hinder further optimizations.

Reply:
At the moment this is intended behaviour to be compatible with the AASRouting pass. After using the smart placement in AAS we could change this.

Reply:
OK, is it worth putting that task on the backlog?

Reply:
I have added that.